frontend/public/components/utils/units.js

```diff
@@ -222,6 +224,24 @@ export const humanizeCpuCores = (v) => {
     value,
   };
 };
+export const humanizeCpuCoresLong = (v) => {
+  const humanized = humanizeCpuCores(v);
+  const transformedUnit = humanized.unit === 'm' ? ' millicores' : ' cores';
+  return {
+    ...humanized,
+    string: `${formatValue(humanized.value)}${transformedUnit}`,
+  };
+};
+export const humanizeCpuCoresCompact = (v) => {
+  // keep k8s values but round to 3 decimal digits
+  const value = Math.round(v * 1000) / 1000;
+  return {
+    string: `${value}`,
+    unit: '',
```

Comment:
So this always returns the value in cores? Why not return millicores and use `m`?

It's confusing when I'd use `humanizeCpuCoresCompact` vs `humanizeCpuCores`. I don't think we need both.

Reply:
The idea was to preserve the `humanizeCpuCores` conforming the internal k8s format, if needed.
And to use `humanizeCpuCoresCompact` for human-facing texts, so with the extra space between the value and the unit.

However, I do not see the `humanizeCpuCores` used in the just described context. So what about just adding the extra space before the `m` in the `humanizeCpuCores`, so we will end up just with this `humanizeCpuCores` function plus `humanizeCpuCoresLong`?